test/pylib/cpp/item.py

```diff
@@ -32,7 +32,7 @@
 from _pytest._code.code import TerminalRepr, ReprFileLocation
 from _pytest._io import TerminalWriter
 
-from test.pylib.cpp.boost.boost_facade import COMBINED_TESTS
+from test import COMBINED_TESTS, BUILD_DIR
```

Comment:
Does this change belong here, not to "test.py: move get_combined_tests to the correct facade "?

Reply:
Yes, it belongs here. This only needed because I'm removing use of the parameter project_root that used to determine the build directory, and therefore I need it here in the line 143.